integration-tests/pom.xml

```diff
@@ -10,6 +10,7 @@
     <name>Quarkus JasperReports - Integration Tests</name>
     <properties>
         <skipITs>true</skipITs>
+        <lombok.version>1.18.34</lombok.version>
```

Comment:
Boo.  Down with this abomination.  :-1:

Reply:
Oh man it makes code so much cleaner I don't do a project without it...

Reply:
Setters and getters are boilerplate waste and i hate writing this...

```java
private static final Logger LOG = Logger.getLogger(Application.class);
```

Reply:
It's the first thing I look for to refactor out.  Worse yet, no lines of code were saved with the logger declaration.  At least it was the JBoss logger, I always use the JUL logger in my projects. :smiling_imp:  Getters are easy, just an IDE shortcut away.

Then there's the comments here - https://github.com/quarkusio/quarkus/discussions/43179

:shrug: c'est la vie